### PR TITLE
Letter c added to the font boomer

### LINKED ISSUE #348: Add character: <c> to font: <boomer>
*Character* : <character>
*Font* : <fontname>

### LINKED ISSUE #350: Add character: <4> to font: <starwar>
*Character* : <character>
*Font* : <fontname>

### PR DESCRIPTION
**Pull request by Harry121199**
*I have added letter c to the font boomer*

### Referenced Issues

closes #348
closes #350

### CODE REVIEW

File: Fonts/Boomer/boomer.h [hunk 1]
```diff
@@ -349,58 +340,55 @@ class Boomer : public Fonts
         return character;
     }
 
-    /*
-        char **c()
-        {
-            char **character = getCharGrid();
+    char **c()
```
Comment (AnshumanMahato) [inline Fonts/Boomer/boomer.h:343]: Adjust the grid so that c aligns with the bottom.

currently, this is happening
image

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/Boomer/boomer.h (hunk 1 above)
Comment (Harry121199) [reply Fonts/Boomer/boomer.h:343]: should it match with the bottom of the A?

Comment (AnshumanMahato) [reply Fonts/Boomer/boomer.h:343]: Yes

Comment (Harry121199) [reply Fonts/Boomer/boomer.h]: Done sir.

**Comment by Harry121199** [comment]
done all the required changes

File: Fonts/Boomer/boomer.h (hunk 1 above)
Comment (AnshumanMahato) [reply Fonts/Boomer/boomer.h:343]: great. Everything works fine 🥳

**Review by AnshumanMahato — APPROVED** [review]

**Pull request merged**